server/rest.http

```diff
@@ -9,17 +9,18 @@ POST http://localhost:3001/auth/register
 Content-Type: application/json
 
 {
-    "email" : "houssem2@exmaple.com",
+    "email" : "houssem@exmaple.com",
```

Comment:
nitpick: noticed we spelled example wrong in the email 😄

Reply:
sorry it's a typo ;)